### PR TITLE
classifiers

### PR DESCRIPTION
**Pull request by jburel**
Add classifiers section
cc @aleksandra-tarkowska @will-moore @sbesson @joshmoore 
see https://pypi.python.org/pypi?%3Aaction=list_classifiers

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -48,6 +48,17 @@ def read_file(fname, content_type=None):
...
       description=DESCRIPTION,
       long_description=read_file('README.rst'),
+      classifiers=[
+          'Development Status :: 5 - Production/Stable',
+          'Intended Audience :: End Users/Desktop',
+          'License :: OSI Approved :: GNU Affero General Public License v3 '
+          'or later (AGPLv3+)',
+          'Natural Language :: English',
+          'Operating System :: OS Independent',
+          'Programming Language :: JavaScript',
```
Comment (aleksandra-tarkowska) [inline setup.py]: aren't we missing Python?

```
'Programming Language :: JavaScript',
Programming Language :: Python
Programming Language :: Python :: 2
Programming Language :: Python :: 2.7
Programming Language :: Python :: 2 :: Only
```

Comment (aleksandra-tarkowska) [inline setup.py]: sure we can, look at https://pypi.python.org/pypi/Django/1.8 :)

Comment (jburel) [inline setup.py]: cool!!

**Comment by aleksandra-tarkowska** [comment]
just for final test could you push to pypi please

**Comment by will-moore** [comment]
What about ```Topic :: Scientific/Engineering :: Visualization``` ?
I'm not sure that this is a 'Software Development' tool?

**Comment by aleksandra-tarkowska** [comment]
There are also 

```
Intended Audience :: Science/Research
Topic :: Scientific/Engineering :: Bio-Informatics
Topic :: Scientific/Engineering :: Visualization
```

**Comment by jburel** [comment]
I think we might have to be more selective, I don't think that it is a tool I will put in the Bio-informatics category

**Comment by jburel** [comment]
@aleksandra-tarkowska @will-moore: if we are happy with the list, I will push to testpypi for a final check

File: setup.py [hunk 2]
```diff
@@ -48,6 +48,28 @@ def read_file(fname, content_type=None):
...
+          'Programming Language :: Python :: 2',
+          'Topic :: Internet :: WWW/HTTP',
+          'Topic :: Internet :: WWW/HTTP :: Dynamic Content',
+          'Topic :: Internet :: WWW/HTTP :: WSGI',
+          'Topic :: Scientific/Engineering :: Visualization',
+          'Topic :: Software Development :: Libraries :: '
+          'Application Frameworks',
+          'Topic :: Software Development :: Libraries :: Python Modules',
+          'Topic :: Text Processing :: Markup :: HTML'
```
Comment (aleksandra-tarkowska) [inline setup.py]: Could they go in order please?

Comment (jburel) [inline setup.py]: ??? they are in order. I had to "break" 67/68" lines due to flake8 issue

**Comment by jburel** [comment]
@aleksandra-tarkowska @will-moore pushed to testpypi as version 2.0.11
if you could have a look

**Comment by will-moore** [comment]
Looks great. Good to merge.

**Pull request merged**